client/src/app/site/pages/meetings/pages/motions/components/motion-forward-dialog/services/motion-forward-dialog.service.ts

```diff
@@ -74,8 +81,18 @@ export class MotionForwardDialogService extends BaseDialogService<MotionForwardD
         const toMeetingIds = (await firstValueFrom(dialogRef.afterClosed())) as Ids;
         if (toMeetingIds) {
             try {
-                const forwardMotions = toForward.map(motion => this.formatService.formatMotionForForward(motion));
+                const motionIds = toForward.map(motion => motion.id);
+                await this.modelRequest.subscribeTo(getMotionForwardDataSubscriptionConfig(...motionIds));
+                await this.modelRequest.subscriptionGotData(
+                    getSpecificMotionSubscriptionName(MOTION_FORWARD_DATA_SUBSCRIPTION, motionIds)
+                );
+                const forwardMotions = toForward.map(motion =>
+                    this.formatService.formatMotionForForward(this.repo.getViewModel(motion.id))
+                );
                 const result = await this.repo.createForwarded(toMeetingIds, ...forwardMotions);
+                this.modelRequest.closeSubscription(
+                    getSpecificMotionSubscriptionName(MOTION_FORWARD_DATA_SUBSCRIPTION, motionIds)
+                );
```

Comment:
While I think this looks okay instead of generating a specific name you could also just add something to the subscription name like here: `https://github.com/OpenSlides/openslides-client/blob/a2230d48e28a70035366f57bb8216366268c93e0/client/src/app/site/pages/meetings/pages/motions/components/motion-multiselect/services/motion-multiselect.service.ts#L184`

The autoupdate worker will handle duplicates by the content of the subscription.